Code/Tools/AssetProcessor/native/AssetManager/AssetCatalog.cpp

```diff
@@ -488,7 +488,8 @@ namespace AssetProcessor
                         return false;
                     }
 
-                    const bool fileExists = fileStateInterface->Exists(sourceAsset.AbsolutePath().c_str());
+                    AssetProcessor::FileStateInfo fileStateInfo;
+                    const bool fileExists = fileStateInterface->GetFileInfo(sourceAsset.AbsolutePath().c_str(), &fileStateInfo) && fileStateInfo.m_absolutePath.compare(sourceAsset.AbsolutePath().c_str()) == 0;
```

Comment:
would it be helpful to add a note here about what is going on?

Reply:
I looked at this while posting the PR and couldn't remember why this change was needed.  I'm going to revert this for now, I vaguely seem to remember this is where I was testing this change to begin with but I don't think this is needed here.